.github/pull_request_template.md

```diff
@@ -5,12 +5,12 @@
 ...
 
 ### Checklist before you submit for review
-- [ ] Make sure there is a PR in the CNDB project updating the Converged Cassandra version
+- [ ] Make sure there is a PR in the CNDB project updating the Converged Cassandra version if your PR is changing the API
```

Comment:
```suggestion
- [ ] Make sure there is a PR in the CNDB project updating the Converged Cassandra version if your PR depends on specific Converged Cassandra changes.
```

Reply:
isn't a similar problem as we have with Stargate-CNDB relation? Stargate uses CNDB coordinator as an impl so we run stargate internal tests against that implementation;

I think the bare minimum would be to compile CNDB against the CC artifact produced in from the feature branch as a step in CC gating